### ISSUE TITLE
analysis idea: re-cluster the SRA based on taxonomy profiles or on jaccard similarity to de novo produce "biomes" that inherit labels from majority in cluster

### ISSUE THREAD

**Issue by taylorreiter** [dib-lab/2022-sra-gather#2]
Along with mislabelled data, there seems to be a lot of NAs or duplicate labels -- like "seawater metagenome" vs. "marine metagenome." Are these basically the same thing? Can we infer more granular structure than the ScientificNames that are given?

**Comment by ctb** [comment]
yes! please just do a PR to update https://github.com/dib-lab/2022-sra-gather/blob/main/categories/mapping.csv!
